Api/index.php

```diff
@@ -0,0 +1,4 @@
+<?php
+require_once __DIR__. '/Core/app.php';
```

Comment:
The "Core" with an uppercase letter suggests this is a namespace, however the namespace is not defined or used.

Reply:
I can tell only the same like above.
Requiring a file does not suggest any namespaces...